### PR TITLE
Update bootstrap server readme and cli help information

### PR DESCRIPTION
**Pull request by qleisan**
While reviewing #446 I observed some missing/incorrect help information.

### CODE REVIEW

**Comment by qleisan** [comment]
(force push due to missed "signoff" first time...)

**Comment by rettichschnidi** [comment]
LGTM

File: README.md [hunk 1]
```diff
@@ -154,3 +157,10 @@ Options are:
  - -l PORT	Set the local UDP port of the Client. Default: 56830
  - -4		Use IPv4 connection. Default: IPv6 connection
 
+### Bootstrap Server example
+ * Create a build directory and change to that.
+ * ``cmake [wakaama directory]/examples/bootstrap_server``
+ * ``make``
+ * ``./bootstrap_server [Options]``
+
+Refer to examples/bootstrap_server/README for more information.
```
Comment (sbernard31) [inline README.md:166]:
```suggestion
Refer to [examples/bootstrap_server/README](https://github.com/eclipse/wakaama/blob/master/examples/bootstrap_server/README) for more information.
```

Should we add a link here ? 
It sounds good to be able to directly click to go to the documentation.
The issue: this is an absolute link which will always link to the master branch. (But I guess most people target master documentation ? :thinking:)

I'm not sure if this is a good idea.

Comment (qleisan) [reply README.md]: Relative links seem to work e.g. 
[examples/bootstrap_server/README](./examples/bootstrap_server/README)

Comment (qleisan) [reply README.md]: Used the github gui to do the edit in my forked repo, got in trouble again with "eca" :(
Anonymous (qleisan@****.noreply) is not covered by the necessary legal agreements in order to proceed.

Comment (qleisan) [reply README.md]: Solving the "eca" issue was a bit of a struggle...now it is done!

Comment (sbernard31) [reply README.md]: :+1:  LGTM. I will squash both commit and integrate it in master.

**Comment by sbernard31** [comment]
Integrated in master (commit 40b209e7df5fd35e34df95d2540521d29a91b6be)
Thx @qleisan :pray:

**Pull request closed without merging**